src/model_config_tests/conftest.py

```diff
@@ -14,19 +14,19 @@ def output_path(request):
     """Set the output path: This contains control and lab directories for each
     test and test output files - e.g. CHECKSUMS
     """
-    path = request.config.getoption('--output-path')
+    path = request.config.getoption("--output-path")
     if path is None:
         # Set default to /scratch/PROJECT/USER/test-model-repro/
-        project = os.environ.get('PROJECT')
-        user = os.environ.get('USER')
-        path = f'/scratch/{project}/{user}/test-model-repro'
+        project = os.environ.get("PROJECT")
+        user = os.environ.get("USER")
+        path = f"/scratch/{project}/{user}/test-model-repro"
```

Comment:
Maybe we should just use `$TMPDIR` environment variable? It resolves to `/scratch/$PROJECT/$USER/tmp` which is fine, and it would make it less NCI-centric for the future.

Reply:
That would require changing the docs too I guess.

Reply:
Oh awesome, I did not know that was a thing, I'll add that in